src/Storage/Storage.Management/help/Set-AzStorageAccountManagementPolicy.md

```diff
@@ -15,39 +15,43 @@ Creates or modifies the management policy of an Azure Storage account.
 ### AccountNamePolicyRule (Default)
 ```
 Set-AzStorageAccountManagementPolicy [-ResourceGroupName] <String> [-StorageAccountName] <String>
- -Rule <PSManagementPolicyRule[]> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ -Rule <PSManagementPolicyRule[]> [-DefaultProfile <IAzureContextContainer>]
```

Comment:
Please revert change of this file. It's not correct. 
@VeryEarly It looks the file will have several repeated parameters when run build-module again. It would be great if you could help to resolve it!